pkgs/zigtools/zls/registry.yaml

```diff
@@ -1,81 +1,109 @@
 # yaml-language-server: $schema=https://raw.githubusercontent.com/aquaproj/aqua/main/json-schema/registry.json
 packages:
-  - type: github_release
+  - type: http
     repo_owner: zigtools
     repo_name: zls
     description: A Zig language server supporting Zig developers with features like autocomplete and goto definition
+    link: https://zigtools.org/
+    version_source: github_tag
```

Comment:
```suggestion
```

Why do you specify `version_source: github_tag`?
All tags of this repository have releases.
https://github.com/zigtools/zls/releases